core/cautils/getter/gcpcloudapi.go

```diff
@@ -0,0 +1,41 @@
+package getter
+
+import (
+	"context"
+
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1"
```

Comment:
Run `go mod tidy` and commit the `go.mod` and `go.sum` files

Reply:
Can you please elaborate? What is the issue with the packages?

Reply:
in all the files which are changed in this PR imports are throwing error "could not import"